lib/solrizer.rb

```diff
@@ -26,9 +26,17 @@ def default_field_mapper=(field_mapper)
       @@default_field_mapper = field_mapper
     end
 
+    def cache(*args)
+      key = args.join
```

Comment:
Is there any reason the args array itself can't be the cache key?  Rather than try to turn it into a string with `join`? That's what I did in my local version of caching, make the args array itself the cache key.

Reply:
Done.